### PR TITLE
clarify import_ docs

### PR DESCRIPTION
**Pull request by bcoca**
##### SUMMARY

alternate to  #28223

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

docs

##### ANSIBLE VERSION

```
2.4
```

### CODE REVIEW

File: docs/docsite/rst/playbooks_reuse_includes.rst [hunk 1]
```diff
@@ -13,17 +13,25 @@ As noted in :doc:`playbooks_reuse`, include and import statements are very simil
 
 Please refer to  :doc:`playbooks_reuse` for documentation concerning the trade-offs one may encounter when using each type.
 
+Also be aware that this behaviour changed in 2.4, prior to that Ansible version only ``include`` was available and behaved differntly depending on context.
```
Comment (agaffney) [inline docs/docsite/rst/playbooks_reuse_includes.rst]: You spelled "differently" wrong

Comment (bcoca) [reply docs/docsite/rst/playbooks_reuse_includes.rst]: im just using alternative spelling!

**Review by dharmabumstead — APPROVED** [review]
Approved with some edits. Thanks @bcoca!

**Comment by abadger** [comment]
chery-picked for 2.4.0rc2

**Pull request merged**